UI/index.html

```diff
@@ -0,0 +1,434 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
```

Comment:
It looks bad on mobile devices. For better adaptivity try to use `<meta name="viewport" content="width=device-width, initial-scale=1.0">`